style.css

```diff
@@ -0,0 +1,59 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: poppins, serif;
+}
+
+main {

```

Comment:
- [ ] please change this fixed width given to main and put 100% so as to make your work responsive. 

![Screenshot (1103)_LI](https://github.com/MargaretTechworld/Portfolio/assets/65298048/438b0a8e-98ac-4314-b0b0-acd7c28b91aa)